src/src/models/MessageModel.ts

```diff
@@ -0,0 +1,5 @@
+export default interface MessageModel {
+    isError: Boolean;
+    message: String;
+    propertyName: [String | null];
+};
```

Comment:
Codacy has a fix for the issue: [Missing semicolon.](https://app.codacy.com/gh/Supershift/Sushi.Mediakiwi.UI/pullRequest?prid=8211123)
```suggestion
};;
```